features/FxA-15-OAuth-Management/README.md

```diff
@@ -0,0 +1,83 @@
+Disconnect OAuth services from FxA
+======================
+
+https://mozilla.aha.io/features/FXA-15
+
+## Problem Summary
+
+As a Firefox Accounts user I want to be able to revoke access to the
+services that I logged into.  
+
+****
+
+## Outcomes
+
+The success criteria of this feature:
+* Being able to view logged in OAuth services in the apps view.
+* Being able to revoke access ("disconnect") a service.
+
+See "Detailed Design" below for more details.
+
+## Hypothesis
+
+We believe that by adding OAuth services and functionality to disconnect the services
+for Firefox Accounts users will improve the quality and security of FxA
+for everybody.
```

Comment:
There's some funky sentence structure going on here that makes this hard to parse, it might need a comma or two at least.